extensions/blocks/opentable/edit.js

```diff
@@ -57,6 +58,28 @@ function OpenTableEdit( {
 
 	const { align, rid, style, iframe, domain, lang, newtab } = attributes;
 
+	// We only want to show the warning about wide display when alignment changes, not on every page load
```

Comment:
Is there a reason for this? I'd rather show the warning all the time (when style is wide and align isn't), so that the user is always aware of it.
Otherwise, they might save the post, come back later and have no idea why it looks wonky.

Reply:
I was just assuming that if they decided to ignore the warning and save it, then they had a reason for doing so, so didn't wanted to be reminded every time they loaded the page ... but you maybe you are correct and we should hassle them every time as it will obviously be broken, so have taken this out for now.